db/migrate/20181204153549_create_repository_maintenance_stat.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRepositoryMaintenanceStat < ActiveRecord::Migration[5.1]
+  def change
+    create_table :repository_maintenance_stats do |t|
+      t.references :repository
+      t.string :category
+      t.string :value # just store it as string and make the whatever is reading the stats serialize it
```

Comment:
Are there anything other than numbers that this could be? If not, why not just always use a float?

Reply:
The initial one I had in mind was the average commit date that we were pulling in.